exporter/awscloudwatchlogsexporter/config.go

```diff
@@ -42,7 +42,7 @@ type Config struct {
 	Tags map[string]*string `mapstructure:"tags"`
 
 	// QueueSettings is a subset of exporterhelper.QueueSettings,
-	// because only QueueSize is user-settable due to how AWS CloudWatch API works
+	// because we want to force the queue to always be enabled
```

Comment:
why not replace it with `exporterhelper.QueueSettings` directly

Reply:
the default is set to true. My 2 cents is that users should know what they are doing by explicitly setting the queue to disabled.
I will go ahead and modify the PR to use exporterHelper.QueueSettings.